README.md

```diff
@@ -38,6 +38,7 @@ The following documents are available:
 | NATS Transport Binding        | [v0.3](https://github.com/cloudevents/spec/blob/v0.3/nats-transport-binding.md) | [master](https://github.com/cloudevents/spec/blob/master/nats-transport-binding.md) |
 | Protocol Buffers Event Format | [v0.3](https://github.com/cloudevents/spec/blob/v0.3/protobuf-format.md)        | [master](https://github.com/cloudevents/spec/blob/master/protobuf-format.md)        |
 | Web hook                      |      [v0.3](https://github.com/cloudevents/spec/blob/v0.3/http-webhook.md)      |      [master](https://github.com/cloudevents/spec/blob/master/http-webhook.md)      |
+| AVRO Event Format             | - | [master](./avro-format.md) | 
```

Comment:
can you move this so the list if alphabetized?

Reply:
@fabiojose can you also add this to the Makefile so it gets verified?

Reply:
@duglin  done!